_materials/4_nanopore_sequencing_extended.html

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
As the content here is mostly text, not slides, we could maybe move it to the Sequencing protocol as a comment for the teachers